src/mainwindow.cpp

```diff
@@ -1005,12 +1005,7 @@ void MainWindow::saveSpecificedPath(QString path)
     QString fileDir = QUrl::fromLocalFile(QFileInfo(savePath).absoluteDir().absolutePath()).toString();
     QString filePath =  QUrl::fromLocalFile(savePath).toString();
     QString command;
-    if (QFile("/usr/bin/dde-file-manager").exists()) {
-        command = QString("/usr/bin/dde-file-manager,%1?selectUrl=%2"
-                          ).arg(fileDir).arg(filePath);
-    } else {
-        command = QString("xdg-open,%1").arg(filePath);
-    }
+    command = QString("xdg-open,%1").arg(filePath);
```

Comment:
The change breaks the old behavior, we can't accept that, maybe you can try `DDesktopServices::showFileItem` from DTK to see if that satisfy your need?

Reply:
Can you explain how this breaks the old behavior?
If I understand correctly, on Deepin (the os) this would still
start `dde-file-manager` because it's default.

Also, any docs on `DDesktopServices::showFileItem`?

Reply:
The original code 
```
QString("/usr/bin/dde-file-manager,%1?selectUrl=%2).arg(fileDir).arg(filePath)
``` 
open the target directory and select the target file. `xdg-open` won't do that.

`DDesktopServices::showFileItem` basically did the same thing.